openapi/components/schemas/Common/CommonOrderPreview.yaml

```
@@ -36,6 +36,10 @@ properties:
     nullable: true
     allOf:
       - $ref: ../Contact/ContactObject.yaml
+  shippingRateId:
+    description: The shipping rate ID. If unset – cheapest shipping rate among applicable would be chosen.
```

```suggestion
    description: The shipping rate ID. If unset the cheapest applicable shipping rate is chosen.
```

https://github.com/Rebilly/api-definitions/pull/644#discussion_r791964287